Library/Homebrew/formula.rb

```diff
@@ -3102,6 +3114,33 @@ def license(args = nil)
       end
     end
 
+    # @!attribute [w] enable_offline_phase
+    # The phases which should be forced offline (network isolation).
+    # By default, all phases are online. Valid phases are `:build`, `:test`, and `:postinstall`.
+    # When no argument is passed, the setting will be enabled for all phases.
+    # <pre>enable_offline_phase</pre>
+    # <pre>enable_offline_phase :build</pre>
+    # <pre>enable_offline_phase [:build, :test]</pre>
+    def enable_offline_phase(phases = [])
```

Comment:
Maybe call it `deny_network_access`? It's a bit more obvious what it does, and implies a natural name for the opposite (`allow_network_access`) for when we start making it the default.

Reply:
+1 on `deny_network_access` -- I think that also makes the default case ("deny all with nothing explcitly listed") a little more intelligible 🙂

Reply:
Yeh, I like that too.

@alebcay I know this seems bizarre but there may be a case (e.g. in `brew tests`) to deny network access during `fetch`!